redash/handlers/users.py

```diff
@@ -94,6 +94,8 @@ def post(self):
         req = request.get_json(force=True)
         require_fields(req, ('name', 'email'))
 
+        if '@' not in req['email']:
+            abort(400, message='Bad email address.!!!')
```

Comment:
Please fix error message (no need for exclamation marks :-) )

Reply:
Wow I forgot to delete the debug message! 😭